frontend/src/components/HomePage.vue

```diff
@@ -17,19 +17,22 @@
           type="error"
           transition="slide-y-transition">
           Błąd w trakcie wysyłania wiadomości
-        </v-alert>
+        </v-alert>     
       </div>
       <app-contact-us/>
+      <app-about-us/> 
```

Comment:
Change the sequence. The component "about us" logically goes before "contact us" :)